bundled/tool/server.py

```diff
@@ -236,7 +236,7 @@ def code_action(params: lsp.CodeActionParams) -> List[lsp.CodeAction]:
     return code_actions
 
 
-@QUICK_FIXES.quick_fix(codes=["E275"])
+@QUICK_FIXES.quick_fix(codes=["274", "E275"])
```

Comment:
This is missing the E
```suggestion
@QUICK_FIXES.quick_fix(codes=["E274", "E275"])
```